packages/util/test/kzg.spec.ts

```diff
@@ -8,7 +8,68 @@ import { getBlobs } from '../src/blobs.ts'
 
 import type { KZG } from '../src/kzg.ts'
 
+import * as ckzg from 'c-kzg'
+import { bytesToHex, hexToBytes } from '../src/bytes.ts'
+
 export const jsKZG = new microEthKZG(trustedSetup)
+const cKzg = {
```

Comment:
all changes here will be reverted when microethsigner will be ready with the new capabilities